test/e2e/sourcery.js

```diff
@@ -14,35 +15,25 @@ module.exports = {
         browser
             .url(browser.launchUrl)
             .waitForElementVisible(common.pageTitle);
-        //Click to expand select user dropdown
-        browser.isVisible(login.userSelect, function(result) {
+        //Select user 
+        browser.isVisible(login.userSelect, function (result) {
```

Comment:
What you get from this code:
        browser.isVisible(login.userSelect, function (result) {
            if (result.status === constants.ELEMENT_FOUND) {

Maybe you can simply remove it?